app/helpers/journey_helper.rb

```diff
@@ -0,0 +1,7 @@
+module JourneyHelper
```

Comment:
This was originally `AdditionalPaymentsHelper` but the remaining code isn't specific to AP